client/my-sites/stats/stats-chart-tabs/index.jsx

```diff
@@ -213,25 +209,6 @@ export default React.createClass( {
 		return chartData;
 	},
 
-	renderNudge: function() {
-		if ( abtest( 'statsTabsLikesNudge' ) === 'dataInformedBelowChart' ) {
-			return (
-				<UpgradeNudge
-					title={ this.translate( 'Sites with Premium get 31% more likes' ) }
```

Comment:
I know we are changing our approach to testing, but lets let current test finish ok?